docs/tax-protocol-graphql-api.md

```diff
@@ -0,0 +1,396 @@
+    <tr>
+      <td colspan="2" valign="top">
+        <strong id="orderformconfiguration.taxconfiguration">taxConfiguration</strong>
+      </td>
+      <td valign="top">
+        <a href="#taxconfiguration">TaxConfiguration</a>
+      </td>
+      <td>External tax service configuration information.</td>
+    </tr>
+    <tr>
+      <td colspan="2" valign="top">
+        <strong id="orderformconfiguration.minimumquantityaccumulatedforitems">
+          minimumQuantityAccumulatedForItems</strong>
+      </td>
+      <td valign="top">
+        Int
+      </td>
+      <td>Minimum SKU quantity by cart. This field is mandatory and must be sent in all requests.</td>
+    </tr>
+    <tr>
+      <td colspan="2" valign="top">
+        <strong id="orderformconfiguration.decimaldigitsprecision">decimalDigitsPrecision</strong>
+      </td>
+      <td valign="top">
+        Int
+      </td>
+      <td>Number of price digits.</td>
+    </tr>
+    <tr>
+      <td colspan="2" valign="top">
+        <strong id="orderformconfiguration.minimumvalueaccumulated">minimumValueAccumulated</strong>
+      </td>
+      <td valign="top">
+        Int
+      </td>
+      <td>Minimum cart value.</td>
+    </tr>
+    <tr>
+      <td colspan="2" valign="top">
+        <strong id="orderformconfiguration.apps">apps</strong>
+      </td>
+      <td valign="top">[<a href="#app">App</a>]</td>
+      <td>List of App objects, where each object has an app configuration information. See an example with the <code>apps</code> list in the <a href="https://developers.vtex.com/docs/guides/get-an-account-orderform-configuration" target="_blank">Get orderForm configuration</a> response body.</td>
+    </tr>
+    <tr>
+      <td colspan="2" valign="top">
+        <strong id="orderformconfiguration.allowmultipledeliveries">allowMultipleDeliveries</strong>
+      </td>
+      <td valign="top">
+        Boolean
+      </td>
+      <td>Allows the selection of items from several delivery channels in the same purchase.</td>
+    </tr>
+    <tr>
+      <td colspan="2" valign="top">
+        <strong id="orderformconfiguration.allowmanualprice">allowManualPrice</strong>
+      </td>
+      <td valign="top">
+        Boolean
+      </td>
+      <td>Allows the editing of SKU prices directly in the cart.</td>
+    </tr>
+    <tr>
+      <td colspan="2" valign="top">
+        <strong id="orderformconfiguration.maxintofwhitelabelsellers">maxIntOfWhiteLabelSellers</strong>
+      </td>
+      <td valign="top">
+        Int
+      </td>
+      <td>Allows the input of a limit of white label sellers involved on the cart.</td>
+    </tr>
+    <tr>
+      <td colspan="2" valign="top">
+        <strong id="orderformconfiguration.maskfirstpurchasedata">maskFirstPurchaseData</strong>
+      </td>
+      <td valign="top">
+        Boolean
+      </td>
+      <td>Allows, on a first purchase, masking client's data. It could be useful when a shared cart is used and the client doesn't want to share its data.</td>
```

Comment:
```suggestion
      <td>Masks customer data for first-time purchases. It could be useful when a shared cart is used, and the client doesn't want to share its data.</td>
```